templates/zerver/api/get-single-user.md

```diff
@@ -0,0 +1,56 @@
+# Get a user
+
+Retrieve a single user in a realm.
```

Comment:
This isn't good user-facing text; API users don't need to know what a "realm" is, since we refer to them as an "organization" in the API.  We should describe this as "Fetch data about a single user in the organization.".

And then we should add a second sentence that says "To fetch users in bulk, see ..." with a link to the bulk-fetch API.